prog/vnet/cert_nexus.rb

```diff
@@ -64,7 +64,7 @@ def before_run
     when "pending"
       nap 10
     when "valid"
-      csr_key = OpenSSL::PKey::RSA.new(4096)
+      csr_key = OpenSSL::PKey::EC.generate("prime256v1")
       csr = Acme::Client::CertificateRequest.new(private_key: csr_key, common_name: cert.hostname)
```

Comment:
this does change our non-test behavior; P256 is common, is it okay with the acme certificaterequest?